src/components/Dashboard.vue

```diff
@@ -529,40 +527,64 @@ export default {
     },
 
     endOnboarding(onboardingData) {
+      console.log(onboardingData);
       const user = this.parseUserData(onboardingData.userData, onboardingData.name);
 
       this.user = user;
       this.loaded = true;
 
       const docRef = this.getDocRef();
-
       const data = {
         name: onboardingData.name,
         userData: onboardingData.userData,
         semesters: this.firebaseSems,
         subjectColors: this.subjectColors
       };
-
       // set the new name and userData, along with either an empty list of semesters or preserve the old list
       docRef.set(data);
-
       this.cancelOnboarding();
       this.updateRequirementsMenu();
     },
 
     cancelOnboarding() {
       this.isOnboarding = false;
     },
-
     parseUserData(data, name) {
       const user = {
         // TODO: take into account multiple majors and colleges
         college: data.colleges[0].acronym,
         collegeFN: data.colleges[0].fullName,
         firstName: name.firstName,
         middleName: name.middleName,
-        lastName: name.lastName
+        lastName: name.lastName,
+        exam: [],
+        transferCourse: [],
+        tookSwim: data.tookSwim
       };
+      const transferClasses = [];
+      if ('exam' in data && data.exam.length > 0) {
+        const exams = [];
+        data.exam.forEach(exam => {
+          exams.push(exam);
+          if ('equivCourse' in exam) {
+            transferClasses.push(exam.equivCourse[0]);
```

Comment:
Is this just adding the first equivalent class? If so, this code might need to change when there are exams that are and/or. Might want to drop a TODO here for that

Reply:
Could you clarify what an and/or for an exam would mean?

Reply:
Yeah! Based on what I can gather from the current examCredits file and the Cornell test info site, I think 'or' would be used if a test can only be used for 1 course in courseEquivalents, while 'and' means it can be used to fulfill all of them. A good example of 'or' would be chemistry as you have to pick between getting credit for CHEM 2070 or 2090, and an example of 'and' would be BC Calc since you can get credit for both calc 1 and calc 2 (although it might get a bit more complicated).

Reply:
Ohhhhh okay I'll add a TODO for that :))